src/components/ModalPayment/ModalPayment.tsx

```diff
@@ -26,44 +26,39 @@ const ModalPayment = ({
   amount,
   sellerId,
 }: Props) => {
-  const payment: PaymentParams = {
-    amount: Number(amount) * 100,
-    currency: 'usd',
-    item_type: purchaseType,
-    quantity: 1,
-    seller_id: sellerId,
-  };
-
-  const purchaseTypePhrase =
-    purchaseType === 'donation' ? 'Donation' : 'Voucher purchase';
-
   const [isShown, setIsShown] = useState(false);
-  const showConfirmModal = () => setIsShown(true);
   const [isChecked, setChecked] = useState(false);
-  const checkAgreement = () =>
-    isChecked ? setChecked(false) : setChecked(true);
   const [name, setName] = useState('');
   const [email, setEmail] = useState('');
 
-  const buyer: Buyer = { name, email };
-
   const stripe = useStripe();
```

Comment:
FYI we deleted Stripe in the backend today. We can do the same in the frontend too

Reply:
if stripe was deleted, then im going to delete the whole integration with stripe (ModalPayment file) and keep Square Modal the way it is

Reply:
nice